src/concepts/the-sane-helm-guide.md

```diff
@@ -222,7 +222,7 @@ metadata:
   [...]
 ```
 
-The output is well known - although lengthy - Kubernetes yaml. These are the yamls Helm is applying on the cluster, this time though it was only rendering text files on your laptop.
+The output is a well known - although lengthy - Kubernetes yaml. These are the yamls Helm is applying on the cluster, this time though it was only rendering text files on your laptop.
```

Comment:
Using `the` instead of `a`? Otherwise this needs fix, agreed.

Reply:
Sure, "the" can work just as well.